LayoutTests/resize-observer/contain-instrinsic-size-should-not-leak-nodes.html

```diff
@@ -77,7 +93,23 @@
     await nextRendering();
     gc();
     await nextRendering();
-    assert_equals(internals.referencingNodeCount(document) < initialNodeCount + CISCount * 0.8, true, 'More than 20% of nodes should be collected');
+
+    let checkCount = 0;
+    let handle = setInterval(() => {
```

Comment:
It's a little awkward to have this in the middle of a promise test. I'd recommend wrapping this interval up in a function returning a promise that can be awaited here.

Then you can either `resolve` or `reject` from within the `setInterval` depending on how the test played out.